lib/sgtn-client/api/translation.rb

```diff
@@ -62,8 +61,8 @@ def self.getStrings(component, locale)
         if items.nil? || items["messages"] == nil
           items = {}
           s = SgtnClient::Source.getSources(component, default)
-          key, value = s.first
-          items["component"] = key
+          default_component, value = s.first
```

Comment:
I remember component name is the folder name of source file. Isn't it?

Reply:
the component name is also defined in the file, e.g.
default:
  helloworld: "Hello world"
  hello: "Hello"

Reply:
Why duplicate this information? Seems not a good pratice.

Reply:
what's your suggestion? if you look at other localized json file, there's a component name defined in the file. I think the duplication is useful to  identify the file's content as belong to which component.

Reply:
Suggest one information can only be determined at one place. Multiple places have same information introduces redundanc, complexity and more effort to maintainance.